docs/docs/reference-react-component.md

```diff
@@ -151,6 +207,28 @@ componentDidMount()
 
 `componentDidMount()` is invoked immediately after a component is mounted. Initialization that requires DOM nodes should go here. If you need to load data from a remote endpoint, this is a good place to instantiate the network request. Setting state in this method will trigger a re-rendering.
 
+If you need to access your element, you can use `ReactDOM.findDOMNode(this)`:
```

Comment:
I think this note is suitable to be in the reference (we can change it to a Note block).